### PR TITLE
[ADD] manage multi-company backend

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
Hi @guewen, could you consider this PR in the migration of the magento connector. It comes from https://github.com/OCA/connector-magento/pull/212 on v8.0.

It ran on production for more than a year so it should be safe.

Kind regards,

### CODE REVIEW

File: connector_magento/__manifest__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
  'version': '10.0.1.0.0',
  'category': 'Connector',
  'depends': ['account',
+             'base_technical_user',
```
Comment (guewen) [inline connector_magento/__manifest__.py:10]: why is it required? I can't find this addon

**Review by guewen — COMMENTED** [review]
Thanks for your PR! I'll definitely integrate it before we merge my PR. Just have one question of the new dependency :)

File: connector_magento/__manifest__.py (hunk 1 above)
Comment (Cedric-Pigeon) [reply connector_magento/__manifest__.py:10]: Sorry it comes from https://github.com/OCA/server-tools/pull/858

It avoids to launch jobs with admin user and uses a dedicated technical company user.
It intends to replace https://github.com/OCA/connector-magento/pull/212/files#diff-131bcecd325b739cf356543a96bb2747R11 in a more generic way.

Comment (guewen) [reply connector_magento/__manifest__.py:10]: OK I see! I added a comment on OCA/server-tools#858

Comment (guewen) [reply connector_magento/__manifest__.py:10]: Do you plan to change something about https://github.com/OCA/server-tools/pull/858#discussion_r125591029?

**Comment by guewen** [comment]
@Cedric-Pigeon do you have the cassette file for the test test_import_partner_individual_2_addresses_multi_company?

**Comment by Cedric-Pigeon** [comment]
Do I have to make a cassette for each test ?  I don't understand well how it works ...

**Comment by guewen** [comment]
The cassette is recorded automatically when the test is run and doesn't error. The other test doesn't need a cassette has it doesn't interact with magento. If you are interested: http://vcrpy.readthedocs.io/en/latest/usage.html this is a great lib to know :)

**Comment by guewen** [comment]
I have the feeling that you could either just add your asserts in the existing test_import_partner_individual_2_addresses or reuse the cassetteof this test with something like @recorder.use_cassette('test_import_partner_individual_2_addresses') (but them import the same partner id)

**Comment by guewen** [comment]
I merged at 8f425de. I changed the test with `@recorder.use_cassette('test_import_partner_individual_2_addresses')` and it works, it reuses the same cassette than the `test_import_partner_individual_2_addresses` test.

**Pull request closed without merging**